BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -226,8 +226,6 @@ extension GiniBankNetworkingScreenApiCoordinator {
 
 
                 self.giniBankConfiguration.lineItems = result.lineItems
-
-                documentService.resetToInitialState()
```

Comment:
[Clarification] where is ` documentService.resetToInitialState()`called?

Reply:
As you suggested:
`documentService.resetToInitialState() needs to be moved at the end of your cleanup method.`